src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -356,6 +357,22 @@ public function write($messages, $newline = false, $type = self::OUTPUT_NORMAL)
     }
 
     /**
+     * @return QuestionHelper|null
```

Comment:
i want to test the action of interactively input:
1. creating a `SymfonyQuestionHelper` object in the constructor for making code simpler.
2. use `getQuestionHelper()` to get the `SymfonyQuestionHelper` object for test interactively input with mock via [setInputStream()](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Helper/QuestionHelper.php#L75)